server/app/services/models/BotResponse.java

```diff
@@ -1,9 +1,9 @@
-package services;
+package services.models;
 
 /**
  * Created by sabinapokhrel on 10/1/16.
  */
-public class Response {
+public class BotResponse {
```

Comment:
Can you please add a simple comment to explain what is a Bot Response here? Like response from `HuBot`?